src/app/shared/components/template/components/dashed-box/dashed-box.component.scss

```diff
@@ -56,4 +56,6 @@
       color: var(--ion-color-secondary);
     }
   }
+  // noticeably, i realised that in the debug_dashed_box component, the class is not viewed
+  // as a paragraph but actually a div class containing text
```

Comment:
You can remove this comment – it could be included in the PR, but we don't want it in the code